src/utils/bridge/eth/classes/reducers.ts

```diff
@@ -74,7 +76,7 @@ export class EthBridgeReducer extends BridgeReducer<BridgeHistory> {
         });
       } catch (error: any) {
         // maybe transaction already completed, try to restore ethereum transaction hash
-        if (error.code === ethers.errors.UNPREDICTABLE_GAS_LIMIT) {
+        if (error.code === 'UNPREDICTABLE_GAS_LIMIT') {
```

Comment:
don't we have a constant somewhere?

Reply:
No, in ethers v6 they are removed. And this error comes from metamask (it was in v5 for some reasons)